### ISSUE TITLE
`opaqueGenericParameters` rule doesn't apply to `init`s

### ISSUE THREAD

**Issue by calda** [nicklockwood/SwiftFormat#1291]
The `opaqueGenericParameters` rule isn't currently being applied to `init` declarations.

This declaration is not currently updated:

```swift
init<Title: StringProtocol>(title: Title) {
  ...
}
```

but should be updated to:

```swift
init(title: some StringProtocol) {
  ...
}
```

**Comment by calda** [comment]
I plan on fixing this, just filing the issue for visibility

**Comment by nicklockwood** [comment]
@calda Is this also applicable to subscripts?

**Comment by calda** [comment]
Ah yes you're right -- I always forget about `subscript`. I updated https://github.com/nicklockwood/SwiftFormat/pull/1292 to cover those as well.

**Comment by nicklockwood** [comment]
@calda fix landed in 0.50.3
